src/genomic_features/annotate_anndata.py

```diff
@@ -0,0 +1,112 @@
+import warnings
+
+import pandas as pd
+
+
+def annotate_anndata(
```

Comment:
Since the function is passed a dataframe and returns a dataframe, what about calling it something like `annotate_dataframe`, `annotate_table`, or even: `annotate`?